### PR TITLE
Add docs for setting the RUNNER_GRACEFUL_STOP_TIMEOUT env var on docker container

### PR DESCRIPTION
**Pull request by nsheaps**
While investigating some instability with our long running builds using ARC, I found that when termination signals were sent, the runner container would terminate gracefully according to the docs, but the docker daemon would not. The cause was that this env var was not set on the docker container. After looking at the Pod and it's preStop hook, the `docker` container is already setup to handle graceful termination, but the documentation was not clear that this env var needed to be set on that container as well.

This PR clarifies the docs and explicitly says where to add the variable

### CODE REVIEW

**Review by mumoshu — APPROVED** [review]
Good catch! Thank you so much for finding and fixing it @nsheaps ☺️

**Comment by nsheaps** [comment]
No problem @mumoshu ! Please merge this when you feel ready, as I don't have permission to merge it.

Also I think your first-interaction action might be missing some permissions for the token, but I see it's not required.

**Comment by mumoshu** [comment]
@nsheaps Thanks for your confirmation! I was just waiting for the CI build to pass and it looks alright now!
Re: the first-interaction workflow- yeah it seems not working but I think it isn't strictly needed.
Let me merge this now!
Thanks again for your contribution ☺️

**Pull request merged**
